app/models/custom_workflow.rb

```diff
@@ -105,6 +104,7 @@ def has_projects_association?
   def validate_syntax_for(object, event)
     object.instance_eval(read_attribute(event)) if respond_to?(event) && read_attribute(event)
   rescue WorkflowError => e
+    errors.add event, :invalid_script, :error => e
```

Comment:
It should not say "Invalid script here"  during validation  because  WorkflowError is considered like message to the user, so it is _not_ an exception

Reply:
You're right.